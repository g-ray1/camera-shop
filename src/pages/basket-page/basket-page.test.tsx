import { render } from '@testing-library/react';
import BasketPage from './basket-page';
import { BrowserRouter } from 'react-router-dom';

jest.mock('react-redux');

describe('BasketPage component', () => {
  it('Should render BasketPage', () => {
    const view = render(<BasketPage />, {wrapper: BrowserRouter});

    expect(view).toMatchSnapshot();
  });
});

