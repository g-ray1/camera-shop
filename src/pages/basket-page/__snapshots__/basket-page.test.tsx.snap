// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BasketPage component Should render BasketPage 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <main>
        <div
          class="page-content"
        >
          <div
            class="breadcrumbs"
          >
            <div
              class="container"
            >
              <ul
                class="breadcrumbs__list"
              >
                <li
                  class="breadcrumbs__item"
                >
                  <a
                    class="breadcrumbs__link"
                    href="/page_1"
                  >
                    Главная
                    <svg
                      aria-hidden="true"
                      height="8"
                      width="5"
                    >
                      <use
                        xlink:href="#icon-arrow-mini"
                      />
                    </svg>
                  </a>
                </li>
                <li
                  class="breadcrumbs__item"
                >
                  <span
                    class="breadcrumbs__link breadcrumbs__link--active"
                  >
                    Каталог
                  </span>
                </li>
              </ul>
            </div>
          </div>
          <div
            class="page-content"
          >
            <div
              class="page-content__section"
            >
              <div
                class="container"
              >
                <h1
                  class="title title--h2"
                >
                  Page Not Found
                </h1>
                <a
                  class="btn"
                  href="/"
                >
                  Вернутся на главную
                </a>
              </div>
            </div>
          </div>
        </div>
      </main>
    </div>
  </body>,
  "container": <div>
    <main>
      <div
        class="page-content"
      >
        <div
          class="breadcrumbs"
        >
          <div
            class="container"
          >
            <ul
              class="breadcrumbs__list"
            >
              <li
                class="breadcrumbs__item"
              >
                <a
                  class="breadcrumbs__link"
                  href="/page_1"
                >
                  Главная
                  <svg
                    aria-hidden="true"
                    height="8"
                    width="5"
                  >
                    <use
                      xlink:href="#icon-arrow-mini"
                    />
                  </svg>
                </a>
              </li>
              <li
                class="breadcrumbs__item"
              >
                <span
                  class="breadcrumbs__link breadcrumbs__link--active"
                >
                  Каталог
                </span>
              </li>
            </ul>
          </div>
        </div>
        <div
          class="page-content"
        >
          <div
            class="page-content__section"
          >
            <div
              class="container"
            >
              <h1
                class="title title--h2"
              >
                Page Not Found
              </h1>
              <a
                class="btn"
                href="/"
              >
                Вернутся на главную
              </a>
            </div>
          </div>
        </div>
      </div>
    </main>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
